README.md

```diff
@@ -4,29 +4,29 @@ Built with [Wercker](http://www.wercker.com)
+Oracle is finding ways for organizations using WebLogic Server to run important workloads, to move those workloads into the cloud. By certifying on industry standards, such as Docker and Kubernetes, WebLogic now runs in a cloud neutral infrastructure. To enhance WebLogic taking full advantage of Kubernetes, you can persist the WebLogic domain home in a Docker image, making deployment much more portable and improving support for CI/CD deployments.
 
-Oracle has been working with the WebLogic community to find ways to make it as easy as possible for organizations using WebLogic Server to run important workloads, to move those workloads into the cloud.  One aspect of that effort is the creation of the Oracle WebLogic Server Kubernetes Operator.  This release of the Operator provides a number of features to assist with the management of WebLogic domains in a Kubernetes environment, including:
+In addition, we've provided an open-source Oracle WebLogic Server Kubernetes Operator (the “operator”) which has several key features to assist you with deploying and managing WebLogic domains in a Kubernetes environment. You can:
 
-*	A mechanism to create a WebLogic domain on a Kubernetes persistent volume. This persistent volume can reside in NFS.
-*	A mechanism to define a WebLogic domain as a Kubernetes resource (using a Kubernetes custom resource definition).
-*	The ability to automatically start servers based on declarative startup parameters and desired states.
-* The ability to manage a WebLogic configured or dynamic cluster.
-*	The ability to automatically expose the WebLogic Server Administration Console outside the Kubernetes cluster (if desired).
-*	The ability to automatically expose T3 channels outside the Kubernetes domain (if desired).
-*	The ability to automatically expose HTTP paths on a WebLogic domain outside the Kubernetes domain with load balancing, and to update the load balancer when Managed Servers in the WebLogic domain are started or stopped.
-*	The ability to scale a WebLogic domain by starting and stopping Managed Servers on demand, or by integrating with a REST API to initiate scaling based on WLDF, Prometheus/Grafana, or other rules.
-*	The ability to publish Operator and WebLogic Server logs into Elasticsearch and interact with them in Kibana.
+*	Create WebLogic domains on a Kubernetes persistent volume. This persistent volume can reside in an NFS.
+* Create a WebLogic domain in a Docker image.
+* Override certain aspects of the WebLogic domain configuration when it is in a Docker image.
```

Comment:
Remove 'when it is in a Docker image'.   (The automatic and custom overrides apply to all domains - not just domain-in-image domains...)

Reply:
done.